testing/src/main/scala/org/http4s/testing/ArbitraryInstances.scala

```diff
@@ -208,7 +208,7 @@ trait ArbitraryInstances {
   def genContentCodingNoQuality: Gen[ContentCoding] =
     Gen.frequency(
       (10, oneOf(ContentCoding.standard.values.toSeq)),
-      (2, Gen.alphaStr.filter(_.nonEmpty).map(ContentCoding.unsafeFromString))
+      (2, genToken.filter(!_.startsWith("*")).map(ContentCoding.unsafeFromString))
```

Comment:
Any `filter` in a generator wrapped in an arbitrary is going to be a potential problem until https://github.com/rickynils/scalacheck/pull/316 gets released.

I think `*foo` is a valid token, and thus a valid content coding.  (A bizarre one, but a valid one.)  I think the problem is in the parser, and this filter should be dropped.